json-schema/registry.json

```diff
@@ -260,6 +260,9 @@
         "enabled": {
           "type": "boolean"
         },
+        "predicate_type": {
+          "type": "string"
+        },
```

Comment:
Do you update this manually?
If so, you can update JSON Schema by `cmdx js` command.
`cmdx js` updates JSON Schema according to Go codes.

Reply:
And CI (autofix.ci) updates JSON Schema automatically, though there is something wrong now.

https://github.com/aquaproj/aqua/pull/3794#issuecomment-2817056054

Reply:
Yes, I did update the schema manually. Thanks for the tips, I'll try to remember them for the future.